testsuite/csv-formatter.cc

```diff
@@ -0,0 +1,40 @@
+
+#define BOOST_TEST_DYN_LINK
+#define BOOST_TEST_MODULE snapper
+
+#include <boost/test/unit_test.hpp>
+
+#include <vector>
+#include <string>
+
+#include "../client/utils/CsvFormatter.h"
+
+using namespace std;
+
+BOOST_AUTO_TEST_CASE(test1)
+{
+    vector<string> columns;
+
+    vector<vector<string>> rows;
+
+    string separator;
```

Comment:
columns, rows and separator can be initialized with the values right way making the code shorter and better readable. Same in JSON test.